pkgs/by-name/li/linien-gui/package.nix

```diff
@@ -4,16 +4,44 @@
   qt5,
 }:
 
-python3.pkgs.buildPythonApplication rec {
+let
+  loosenNumpy =
+    p: pattern:
+    p.overridePythonAttrs (old: {
+      postPatch =
+        (old.postPatch or "")
```

Comment:
```suggestion
        old.postPatch or ""
```

Reply:
Are you sure that:

```nix
old.postPatch or "" + ''
  # more here
''
```

Won't be evaluated like:

```nix
old.postPatch or ("" + ''
  # more here
'')
```

?

Reply:
Not 100% sure. Is it?

Reply:
I'm not motivated enough to check, and this doesn't matter, as the `postPatch` hooks are modified not in a `packageOverrides` now.

Reply:
I just checked using `nix repl` and it woukd work without parentheses.